cluster/cloud-provider.go

```diff
@@ -16,7 +16,7 @@ import (
 const (
 	CloudConfigDeployer    = "cloud-config-deployer"
 	CloudConfigServiceName = "cloud"
-	CloudConfigPath        = "/etc/kubernetes/cloud-config"
+	CloudConfigPath        = "/etc/kubernetes/cloud-config.json"
```

Comment:
We are not using json for each cloud provider now, for vsphere and openstack we use .gcfg extension, so its better to remove the extension altogether